internal/delivery/http/payment_handler.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/kodinggo/payment-service-gb1/internal/model"
 	"github.com/kodinggo/payment-service-gb1/internal/usecase"
 	"github.com/kodinggo/payment-service-gb1/internal/utils"
+	"github.com/kodinggo/user-service-gb1/middleware-example/helper"
```

Comment:
```suggestion
	"github.com/kodinggo/payment-service-gb1/internal/helper"
```

mas salahnya disini mas, auto importnya malah ngarahin ke user service, itu aku suggest import yg bener nya